CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.2] - 2019-07-22
+
+### Fixed
+
+- Index new package only in package-and-push command.
+
```

Comment:
You can test it using `dev:index-new-package-only` version. See https://github.com/giantswarm/architect-orb#development.

Also remember to bump the version: https://github.com/giantswarm/architect-orb#releases

Reply:
@kopiczko `dev:index-new-package-only` cannot be found https://circleci.com/gh/giantswarm/prometheus-operator-app/24
I cannot find that `orb-tools/publish-dev` was triggered.

Reply:
Found it https://circleci.com/gh/giantswarm/architect-orb/219 branch name is not used for the tag - alpha tag and tag with first 8 digits of SHA are published.

Reply:
Thanks @kopiczko for tip on how to test the orb changes - uncovered that `helm package` step fails at least with these modifications because `helm init` has to be called first for helm to guarantee success of commands https://github.com/helm/helm/issues/1732#issuecomment-268093699